mininet/node.py

```diff
@@ -866,10 +868,19 @@ def setHID( self, *hid ):
             self.cmd( 'xip hid new', p )
             self.cmd( 'xip hid addaddr', p )
 
+    def setZFID( self, *zfid ):
+        """Configure zFilter Identifier(s) for the host
+           zfid: A list of zFilter IDs"""
+        moduleDeps( add='xia_ppal_zf' )
+        for z in zfid:
+            if len( z ) == 40 and int( z, 2 ) > 0:
+                self.cmd( 'xip zf addlocal', z )
```

Comment:
If this is not a valid zFilter ID, we should add an error message in an `else` case. You can use the `error()` function.

Reply:
An exception could be useful here but the other day I tried some absurd values for ip like 
`addHost('xia1', cls=XIAHost, ip='1566726')` and all that happened was the host started with no ip configuration. However, I think I will do raise an exception here.